project/src/main/webapp/directory.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta name="Voter Central Candidate Directory Page"
```

Comment:
Would it make more sense to call this a "election directory page" instead of "candidate", since it's not really surfacing individual candidates at this point?